adapters/appnexus/appnexus_test.go

```diff
@@ -30,6 +31,13 @@ func TestMemberQueryParam(t *testing.T) {
 	}
 }
 
+func TestBuilder(t *testing.T) {
```

Comment:
Please consider updating the name of this function to something like `TestBuilderWithPlatformID` as we're specifically trying to test a builder where we're passing a custom platform ID.

Also please add assertion for `adapter.hbsource` to ensure platform ID is correctly set. You can do so as follows:
```
assert.NotNil(t, bidder)
assert.Equal(t, 3, (*bidder.(*adapter)).hbSource)
```

Reply:
Updated assertions in [9a7e773](https://github.com/prebid/prebid-server/pull/2726/commits/9a7e7738aa835c0af2cf07f4195d827fe635fe82)